build.gradle

```
@@ -43,7 +51,7 @@ dependencies {
 
 javafx {
     version = "17.0.2"
-    modules = [ 'javafx.controls' ]
+    modules = [ 'javafx.controls','javafx.media','javafx.fxml', 'javafx.web']
```

```suggestion
    modules = [ 'javafx.base', 'javafx.controls', 'javafx.graphics', 'javafx.fxml', 'javafx.web', 'javafx.media' ]
```
If we're matching Samples we should have all 6